eidas_node/tests/proxy_service/test_views.py

```diff
@@ -298,6 +325,76 @@ def test_create_light_token(self, uuid_mock: MagicMock):
         self.assertEqual(token.encode('sha256', 'test-secret').decode('ascii'), encoded_token)
         self.assertEqual(uuid_mock.mock_calls, [call()])
 
+    @override_settings(PROXY_SERVICE_TRANSIENT_NAME_ID_FALLBACK=False)
+    def test_rewrite_name_id_disabled(self):
```

Comment:
I am missing a test for an unknown format being rewritten as well.

Reply:
Added.